.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+## Description
+
+## Info Required
+
+- Which version of the library do you actually use?
+
+- Do you have the localization permission granted?
+
+- Are you sending parameters to the activity through the bundle?
+
+- Could you describe what are the actions do you make to raise that error?
+
+- Android monitor shows any log related to the library when the error is shown?
```

Comment:
maybe, but the exceptions include the activity class name for the log. For the rest there are no logs or stack traces